src/mgr/DaemonServer.cc

```diff
@@ -787,8 +787,7 @@ bool DaemonServer::_handle_command(
   string prefix;
   cmd_getval(cct, cmdctx->cmdmap, "prefix", prefix);
 
-  dout(4) << "decoded " << cmdctx->cmdmap.size() << dendl;
-  dout(4) << "prefix=" << prefix << dendl;
+  dout(4) << "decoded-size=" << cmdctx->cmdmap.size() << " prefix=" << prefix  << dendl;
```

Comment:
@sebastian-philipp i agree with you that we can s/4/20/ in the previous revision of this PR.

sorry for the bike-shedding. i was wrong.

Reply:
sure enough